CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - All "/api/monkey_control" endpoints to "/api/monkey-control". #1888
 - All "/api/monkey" endpoints to "/api/agent". #1888
 - Update MongoDB version to 4.4.x. #1924
+- Endpoint to get agent binaries from "/api/agent/download/<string:os>" to
```

Comment:
I don't think this is relevant to the users

Reply:
Besides being an API endpoint which seems relevant enough to me, it is encountered by the user if they try to run the monkey manually using the command given in the UI. I feel like it is.

Reply:
Changes for any user-facing interface belong in the changelog.